it.lproj/Localizable.strings

```diff
@@ -0,0 +1,446 @@
+/* 
+  Localizable.strings
+  Palaver
+
+  Created by Alberto Córvo on 18/06/2019.
+  Copyright (c) 2019 Cocode. All rights reserved.
+*/
+
+"REQUIRED" = "Richiesto";
+"OPTIONAL" = "Opzionale";
+"CANCEL" = "Annulla";
+"CLOSE" = "Chiudi";
+"OK" = "OK";
+"YES" = "Yes";
+"NO" = "No";
+"UNKNOWN" = "Sconosciuto";
+
+"NICKNAME" = "Nickname";
+
+/* Body of notification when message previews are disabled */
+"NOTIFICATION_MESSAGE_CATEGORY_PRIVATE_BODY" = "Messaggio";
+
+/* Reply button title for message notification */
+"NOTIFICATION_MESSAGE_CATEGORY_REPLY_ACTION_TITLE" = "Rispondi";
+
+/* Title of notification when sending a message from a notification failed */
+"NOTIFICATION_MESSAGE_CATEGORY_FAILED_TITLE" = "Invio messaggio fallito";
+
+/* Body of notification when sending a message from a notification failed */
+"NOTIFICATION_MESSAGE_CATEGORY_FAILED_BODY" = "Apri applicazione per riprovare";
+
+"NOTIFICATION_REPLY_FAILED" = "Invio messaggio fallito '%@'. Rete IRC Sconosciuta o Cancellata";
+
+"QUERY_LIST_TITLE" = "Chats";
+"QUERY_LIST_TITLE_UNREAD" = "Chats (%@)";
+"QUERY_LIST_ADD_NETWORK_TITLE" = "Aggiungi una rete";
+"QUERY_LIST_ADD_NETWORK_MESSAGE" = "Clicca qui per aggiungere la tua prima rete!";
+
+/* Accessibility label for connected network status */
+"QUERY_LIST_NETWORK_CONNECTED_STATUS" = "Connesso";
+
+/* Accessibility label for disconnected network status */
+"QUERY_LIST_NETWORK_DISCONNECTED_STATUS" = "Disconnesso";
+
+/* Accessibility label for connecting network status */
+"QUERY_LIST_NETWORK_CONNECTING_STATUS" = "Connessione in corso";
+
+/* Accessibility Label for query */
+"QUERY_LIST_QUERY_LABEL" = "%@ (%@ non letto)";
+
+/* Accessibility Label for query with unread mentions */
+"QUERY_LIST_QUERY_UNREAD_MENTIONS_LABEL" = "%@ (%@ non letto, con menzione)";
+
+"QUERY_LIST_SEARCH" = "Cerca messaggi";
+"QUERY_LIST_MARK_AS_READ" = "Segna come già letto";
+
+"NETWORK_ADD" = "Aggiungi Rete";
+"CHANNEL_ADD" = "Iscriviti al canale";
+"NICK_ADD" = "Scrivi ad Utente";
+"NICK_ADD_DONE" = "Messaggio";
+"NICK_ADD_TITLE" = "Scrivi ad Utente";
+"ADD_NICK_SECTION_TITLE" = "Utente";
+
+"NICK_SEND_MESSAGE" = "Manda messaggio";
+"NICK_CHANNEL_KICK" = "Escludi";
+"NICK_CHANNEL_BAN" = "Banna";
+"NICK_CHANNEL_OP" = "Lascia";
+"NICK_CHANNEL_VOICE" = "Lascia vocale";
+"NICK_CHANNEL_DEOP" = "Aderisci";
+"NICK_CHANNEL_DEVOICE" = "Prendi vocale";
+"NICK_IGNORE" = "Ignora";
+"NICK_UNIGNORE" = "Smetti di ignorare";
+
+"CHANNEL_ADD_CHANNEL_NAME" = "Nome";
+"CHANNEL_ADD_PLACEHOLDER_CHANNEL" = "#palaver";
+
+"NETWORK_SERVER_SECTION" = "Server";
+"NETWORK_AUTH_SECTION" = "Identità";
+
+"NETWORK_TITLE" = "Rete";
+"NETWORK_TITLE_PLURAL" = "Reti";
+"NETWORK_NAME" = "Nome";
+"NETWORK_ADDRESS" = "Indirizzo";
+"NETWORK_ADDRESS_PLACEHOLDER" = "Hostname o IP";
+"NETWORK_PORT" = "Porta";
+"NETWORK_USE_TLS" = "SSL";
```

Comment:
"SSL" is used here though "TLS" is used in "TLS_UNTRUSTED_CERTIFICATE_BODY" and "WHOIS_SESSION_USING_TLS"